00-Login/README.md

```diff
@@ -64,27 +49,13 @@ Open the `android/app/src/main/AndroidManifest.xml` file and locate the Intent F
   android:scheme="${applicationId}" />
 ```
 
-> The `applicationId` will get auto-completed dynamically.
-
-### Whitelist the URLs
```

Comment:
Combined this with the Auth dashboard setup above so it all happened in one place.